marlowe/src/Language/Marlowe/Scripts.hs

```diff
@@ -74,7 +82,7 @@ mkMarloweStateMachineTransition params SM.State{ SM.stateData=MarloweData{..}, S
     (interval@(minSlot, maxSlot), inputs) = do
     let positiveBalances = validateBalances marloweState ||
             -- Avoid creating a too-big string literal
-            P.traceError ("Invalid contract state. " `P.appendString` "There exists an account with non positive balance")
+            traceError ("M1")
```

Comment:
Eventually, we might want to create a markdown file that lists the trace error message abbreviations and their interpretations.